Added create new comment

dwb264 opened the pull request.

steveleee commented:
How can I test this out to see if it works?

steveleee commented on dispatch_app/views.py:
```diff
@@ -8,6 +9,26 @@
+@api_view(['POST'])
+def comment(request, tweet_id):
+    try:
+        tweet = Tweet.objects.get(tweet_id=tweet_id)
```
I think this line throws the exception, do we still need the next line?

steveleee commented on dispatch_app/views.py:
```diff
@@ -8,6 +9,26 @@
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    except Tweet.DoesNotExist:
+        return Response({"error": "Tweet does not exist"}, status=status.HTTP_204_NO_CONTENT)
```
Could you make this error message more descriptive? It should help users of the API debug their requests. Following that, I think `HTTP_400` would be more appropriate here.

steveleee commented on dispatch_app/views.py:
```diff
@@ -8,6 +9,26 @@
+@csrf_exempt
+@permission_classes([IsAuthenticated])
+@api_view(['POST'])
+def comment(request, tweet_id):
```
Docs for this function please

dwb264 commented:
I made the following changes:

- Added a test case for creating a comment with an existing and non-existing tweet_id
- Removed the unnecessary line in comment function
- Added description of comment function
- Made comment function error more specific and changed error code to 400

steveleee commented:
This is simply awesome. 👍
